code/modules/mob/living/carbon/human/species_types/vampire.dm

```diff
@@ -45,7 +45,7 @@
 		C.adjustOxyLoss(-4)
 		C.adjustCloneLoss(-4)
 		return
-	C.blood_volume -= 0.75
+	C.blood_volume -= 0.15
```

Comment:
Under the current blood drain, it'll take roughly 19 (excluding server lag) minutes for a fresh 100% blood volume vampire to die. With this change it'd take a whooping 1 hour and 37 minutes.

Vampires bleed from traumas or reagents etcetera just like they replenish through standard methods such as IV bags, iron or bloody mary, and that's fine.

The real deal is how normal low blood symptoms still affect vampires, which can make the race slighty irritating without stockpiling blood beforehand.

I'm cool with reducing the drainage rate to a slighty more sustainable value up to 0.5 (drink iron and forget), but not making it so banal just because of whatever reason you did not provide in your PR description.